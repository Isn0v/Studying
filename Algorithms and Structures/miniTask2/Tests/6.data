34562346324 2143125628