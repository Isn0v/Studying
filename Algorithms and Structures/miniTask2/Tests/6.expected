74071450170775991472